src/hott/07-fibers.rzk.md

```diff
@@ -63,31 +63,31 @@ of the form `#!rzk (a, refl : f a = f a) : fib A B f`.
   ( A B : U)
   ( f : A → B)
   ( C : (b : B) → fib A B f b → U)
-  ( s : (a : A) → C (f a) (a, refl))
+  ( s : (a : A) → C (f a) (a , refl))
   ( b : B)
-  ( (a, q) : fib A B f b)
-  : C b (a, q)
+  ( ( a , q) : fib A B f b)
+  : C b (a , q)
   :=
-    ind-path B (f a) (\ b p → C b (a, p)) (s a) b q
+    ind-path B (f a) (\ b p → C b (a , p)) (s a) b q
 
 #def ind-rev-fib
   ( A B : U)
   ( f : A → B)
   ( C : (b : B) → rev-fib A B f b → U)
-  ( s : (a : A) → C (f a) (a, refl))
+  ( s : (a : A) → C (f a) (a , refl))
   ( b : B)
-  ( (a, q) : rev-fib A B f b)
-  : C b (a, q)
+  ( ( a , q) : rev-fib A B f b)
```

Comment:
But why? There aren't any multi-character tokens before the second parenthesis

Reply:
Well...

Reply:
The point of the space is to help highlight the tree structure of definitions. It is not doing that here. Somewhere along the way, this point lost its significance, and conventions like adding spaces in the type signatures for code alignment were added. I consider that a different convention, although they are easy to conflate.

Reply:
If the code spanned multiple lines, i.e.
```text
  ( ( my-incredibly-long-variable-name ,
      my-other-incredibly-long-variable-name) :
    rev-fib A B f b)
```
_then_ the space should be there because it helps highlight the tree structure.

Reply:
But isn't it theoretically possible to perform deeply nested pattern matching, which one might want to wrap to the next lines?

Reply:
Oh, but with the current conventions this should be:
```suggestion
  ( ( my-incredibly-long-variable-name
    , my-other-incredibly-long-variable-name)
    : rev-fib A B f b)
```

Reply:
> But isn't it theoretically possible to perform deeply nested pattern matching, which one might want to wrap to the next lines?
> 
> **Edit**: Ah, your other comment just loaded now 😅. The thing is that it is not easy to determine beforehand whether the code will span multiple lines, so the space is added anyway for consistency

Although I can sympathize with your situation, I think it is unfortunate to introduce conventions based on the implementation details of the autoformatter.

Reply:
I agree with you for sure, but then does that also apply to the "tree structure" rule everywhere? In other words, should there never be a space after an opening parenthesis unless it is part of a multi-line expression with some nested expressions?

Reply:
It also applies at the leaves

Reply:
But the conventions regarding where to place such spaces have taken on a life of their own, and I don't know what others in the community want.